src/main/java/org/testng/internal/MethodHelper.java

```diff
@@ -435,6 +436,35 @@ public static void fixMethodsWithClass(
     return methodInstances.stream().map(IMethodInstance::getMethod).collect(Collectors.toList());
   }
 
+  public static void dumpInvokedMethodInfoToConsole(
+      ITestNGMethod[] methods, int currentVerbosity) {
+    if (currentVerbosity < 3) {
+      return;
+    }
+    System.out.println("===== Invoked methods");

```

Comment:
Remove sysout

Reply:
@juherr - That was existing code. Its not something I added newly. Will clean up them (found some other places also) in a subsequent PR